lib/utils/utils.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:pispapp/models/account.dart';
+
+class Utils {
+  static String getSecretAccountNumber(Account acc) {
+    final String accountNumber = acc.accountNumber;
+    final StringBuffer sb = StringBuffer();
+    for (int i = 0; i < accountNumber.length; i++) {
+      if (i >= 4 && i <= accountNumber.length - 3) {
+        sb.write('*');
+      } else {
+        sb.write(accountNumber[i]);
+      }
+    }
+    return sb.toString();
+  }
+
+  static String getSecretAccountNumberFromString(String accountNumber) {
```

Comment:
I need both of them. I call this second function on a transactions from/to account number. We don't have an account object in such case.

Reply:
Yeah, I mean, in that case, you can just pass in the account number since that's all you need for both functions to work. You can remove the previous function and update the function call to not pass the whole account. Just a suggestion to avoid code duplication.